src/services/data-service.js

```diff
@@ -0,0 +1,46 @@
+import { Post } from '../models/post';
+
+import firebaseReference from "../firebaseConfig"
+import 'firebase/firestore';
+
+export class DataService {
+	constructor(numPosts) {
```

Comment:
What is numPosts for? Shouldn't the number of posts we get depend on the number of posts that have been saved in the db?

Reply:
Good catch, I actually don't think we need this at all.